devicetypes/smartthings/cree-bulb.src/cree-bulb.groovy

```diff
@@ -93,7 +93,7 @@ def ping() {
 }
 
 def refresh() {
-    zigbee.onOffRefresh() + zigbee.levelRefresh() + zigbee.onOffConfig() + zigbee.levelConfig()
+    zigbee.onOffRefresh() + zigbee.levelRefresh() + zigbee.onOffConfig(0, 300) + zigbee.levelConfig()
```

Comment:
The `cree-bulb` DTH is updated without `levelConfig` & `onOffConfig` both in configure() and refresh()